src/services/variableHelpers.ts

```diff
@@ -85,3 +87,15 @@ export const operatorFunction = function (variable: AdHocFiltersVariable) {
 
   return operators;
 };
+
+/**
+ * For embedded contexts, return the first label as primary label
+ */
+export function getPrimaryLabelFromScene(scene: ServiceScene) {
```

Comment:
This should only be called when embedded, maybe we should gate on the service scene embedded state?